scanner_test.go

```diff
@@ -353,6 +353,64 @@ func TestScan(t *testing.T) {
 			),
 		},
 
+		{
+			name: "oct null",
+			text: "\"\\0\"",
+			want: []byte{
+				0x00,
+			},
+		},
+		{
+			name: "oct null 8",
+			text: "\"\\08\"",
+			want: []byte{
+				0x00, 0x38,
+			},
+		},
+		{
+			name: "oct double",
+			text: "\"\\13\"",
+			want: []byte{
+				0x0b,
+			},
+		},
+		{
+			name: "oct double X",
+			text: "\"\\13" + "X\"",
+			want: []byte{
+				0x0b, 0x58,
```

Comment:
For values here that are obviously ASCII characters, I would prefer a rune literal like 'X'.

Reply:
Done.